### PR TITLE
Fix documentation bugs

### LINKED ISSUE #293: [PE-D][Tester D] The example add in the user guide results in the error message
I copied the example provided in the user guide and pasted it into the add command box, it resulted in an error.

Expected output: Contact successfully added
Actual output: Error message 

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: hooyushing/ped#2

### LINKED ISSUE #368: [PE-D][Tester C] Incomplete sections in DG
1. The Data archiving part is incomplete

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: hm-yang1/ped#8

### LINKED ISSUE #369: [PE-D][Tester C] Justify known issues
Maybe there could be some explanation or justification for known issues? Why is this a feature, not a bug?

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: hm-yang1/ped#9

### LINKED ISSUE #373: [PE-D][Tester D] Lack of support for empty fields
The wedding planner does not support adding of empty fields, and this would pose a problem, as I may not know the phone number, address or email of the person that I would like to add, which would cause some inconvenience if I am unable to fill in all fields

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: hooyushing/ped#26

### LINKED ISSUE #374: [PE-D][Tester B] No editing wedding feature
Currently there is no edit feature for the wedding book, hence if user want to modify some wedding's information, they only can delete the whole wedding and do it again. Maybe could include this in the UG.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: JoanneHing/ped#16

### LINKED ISSUE #375: [PE-D][Tester B] Tag with same wedding
multiple tag with the same wedding can be added but will only detected as one. But this is not mention in the UG.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: JoanneHing/ped#17

### PR DESCRIPTION
**Pull request by zzibo**
fixes #293 
fixes #375 
fixes #374
fixes #373
fixes #369
fixes #368

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -69,23 +69,33 @@ Command Line Interface (CLI) allows you to type text commands to perform specifi
...
+| **[Deleting Tags](#deleting-tags-from-a-contact-tag-del--td)**     | `tag-del n/NAME t/TAG...` or `td n/NAME t/TAG...` <br> e.g., `tag-del n/John Doe t/June & James` or `td n/John Doe t/June & James`                                                          |
+| **[Filter](#filtering-contacts-by-name-and-job-filter--fil)**      | `filter n/KEYWORD` or `filter j/KEYWORD` or `fil n/KEYWORD` <br> e.g., `filter n/John` or `filter j/Photographer` or `fil n/Harry`                                                          |
+| **[View Wedding](#view-wedding-view-wed--vw)**                     | `view-wed NAME & NAME` or `vw NAME & NAME` <br> e.g., `view-wed John & Sarah` or `vw John & Sarah`                                                                                          |
+| **[Clear](#clearing-all-entries-clear-ab--cab-and-clear-wb--cwb)** | `clear-ab` or `cab` for address book and `clear-wb` or `cwb` for wedding book followed by `y` or `n`                                                                                        |
```
Comment (ywllowsensor) [inline docs/UserGuide.md:86]: i think one of the bugs reported said that its abit unclear why we categorise these 2 commands tgt. maybe can separate out to `clear contacts` and `clear weddings`

**Review by ywllowsensor — COMMENTED** [review]
just a minor fix with the clear command in the command summary table

File: docs/UserGuide.md (hunk 1 above)
Comment (zzibo) [reply docs/UserGuide.md:87]: Because our features section groups all the clear command together so if we change this we will have to change that as well

**Comment by jonushzw** [comment]
LGTM!

**Pull request merged**